web/src/components/Plugin/PluginPage.tsx

```diff
@@ -280,11 +284,10 @@ const PluginPage: React.FC<Props> = ({
         const newPlugins = {
           ...initialData,
           [name]: { ...monacoData, disable: !formData.disable },
+          ...(shouldDelete ? { [name]: null } : {}),
```

Comment:
As this component is used in a number of places, we need to be extra careful in modifying this part of the code